environs/manual/sshprovisioner/package_test.go

```diff
@@ -0,0 +1,10 @@
+// Copyright 2013 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// +build linux
```

Comment:
@kat-co @hoenirvili I don't think we should be skipping everything (anything?) in that package on Windows. This code is run from the client, so it shouldn't matter what OS it is. We should be fixing the tests to be cross-platform, rather than not building them. So please use skip so it's in our faces.